editor.planx.uk/src/pages/FlowEditor/components/Settings/index.tsx

```diff
@@ -95,6 +95,7 @@ const Root = styled(Box)(({ theme }) => ({
   left: 0,
   right: 0,
   minHeight: `calc(100% - ${HEADER_HEIGHT}px)`,
+  zIndex: "2000",
```

Comment:
We don't do this consistently anywhere yet, but ideally we'd assign these at the theme level to avoid conflicts.

Might be a nice quick tidy up to pick up at some point?

Docs: 
- https://mui.com/material-ui/customization/z-index/
- https://mui.com/material-ui/customization/default-theme/?expand-path=$.zIndex

Reply:
Ah yes, this is a leftover shim from the previous incarnation of the help text button, which was using fixed positioning and was being pushed above the setting panel. 

I'll remove this for now and look to update the z-index hierarchy at the theme level in a separate PR.